core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -1787,7 +1795,8 @@ public boolean testClassLoad(final String tableName, final String className,
     try {
       return ThriftClientTypes.CLIENT.execute(context,
           client -> client.checkTableClass(TraceUtil.traceInfo(), context.rpcCreds(), tableName,
-              className, asTypeName));
+              className, asTypeName),
+          rgid -> true);
```

Comment:
This could be put in a method, so it can have a better name:

```suggestion
          ResourceGroupId::ANY;
```

Implementation would look something like:

```java
  public static boolean ANY(ResourceGroupId rgid) {
    return true;
  }
```

or

```java
  public static final Predicate<ResourceGroupId> ANY = x -> true;
```

Reply:
Looks good... but ResourceGroupPredicate.DEFAULT looks like "some default predicate implementation" rather than "the resource group named DEFAULT".
I suggest "ResourceGroupPredicate.DEFAULT_RG_ONLY" or something like that.